torch/_prims/context.py

```diff
@@ -230,6 +230,21 @@ def _is_var_mean(self, func):
             and "aten.var_mean" in str(func)
         )
 
+    def _is_view_or_reshape(self, func):
+        return (
+            "torch.Tensor.view" == torch.overrides.resolve_name(func)
+            or "torch.Tensor.reshape" == torch.overrides.resolve_name(func)
+            or "torch.view_copy" == torch.overrides.resolve_name(func)
+            or "torch.reshape" == torch.overrides.resolve_name(func)
+            or (
+                (
+                    isinstance(func, torch._ops.OpOverload)
+                    or isinstance(func, torch._ops.OpOverloadPacket)
+                )
+                and "aten.view" in str(func)
```

Comment:
The last entry is really general. I think we'll also match `torch.ops.aten.view.dtype` here, which takes a slightly different signature than what ` torch.ops.nvprims.view` supports.

A quick regex search with `func: view` in `native_functions.yaml` shows a bunch other ops in the same situation. Is there some other pieces in the stack that protects us from the overloads?

Reply:
I will change this to be specific.